_posts/2018-02-27-django-k8s-elb-health-checks.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: Django, K8s, and ELB Health checks
+author: Paul McLanahan
+excerpt_separator: <!--more-->
+---
+
+As you may have seen in several of our
+[SRE](https://mozilla.github.io/meao/2018/02/16/sre-status/)
+[status](https://mozilla.github.io/meao/2018/01/23/sre-status/)
+[reports](https://mozilla.github.io/meao/2017/12/05/sre-status/)
```

Comment:
probably needs a comma

Reply:
The sentence is just "...several of our SRE status reports." I've just linked the last 3 words.

Reply:
Ohh... nm.. you mean after "reports". Good catch. Done.